src/components/learner-credit-management/tests/AssignmentDetailsTableCell.test.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
[inform] Currently, the tests for the LCM budget detail page are largely integration-style testing rather than testing each sub-component individually. See the failing test from this PR in the `BudgetDetailPage.test.jsx` file where there is currently an assertion on "View course" that would be replaced with your changes (notably the `renders with assigned table data` test case).